example/javascriptlib/basic/6-client-server-realistic/build.mill

```diff
@@ -10,7 +10,7 @@ object server extends JestModule {
   def npmDevDeps =
     super.npmDevDeps() ++ Seq("@types/supertest@^6.0.2", "supertest@^7.0.0")
 
-  override def testConfigSource = Task.Source(millSourcePath / "jest.config.ts")
```

Comment:
`Task.Source` here is correct I think; otherwise `--watch` won't properly notice any changes when `jest.config.ts` changes